crates/ruff/src/args.rs

```diff
@@ -132,13 +132,37 @@ pub enum Command {
     Format(FormatCommand),
     /// Run the language server.
     Server(ServerCommand),
+    /// Analyze the import graph of the given files or directories.
+    #[clap(subcommand)]
+    Graph(GraphCommand),
```

Comment:
I'm leaning toward introducing a more general `analyze` subcommand and make `graph` or `module-graph` a subcommand of that so that we can introduce more analysis in the future without "polluting" the root commands.

Reply:
Sounds good, I like that.